hub/saml-engine/src/integration-test/java/uk/gov/ida/integrationtest/hub/samlengine/apprule/CountryMatchingServiceRequestGeneratorResourceTest.java

```diff
@@ -58,6 +72,16 @@ public void shouldCreateAttributeQueryRequest() throws Exception {
         assertThat(attributeQueryContainerDto.getSamlRequest()).contains("saml2p:AttributeQuery");
     }
 
+    @Test
+    public void shouldCreateAttributeQueryRequestUnsigned() {
+        EidasAttributeQueryRequestDto eidasAttributeQueryRequestDto = new EidasAttributeQueryRequestDtoBuilder()
+                .withAnEncryptedIdentityAssertion()
+                .withACountrySignedResponseWithIssuer("a country entity id")
+                .build();
+        Response response = generateEidasAttributeQueryRequest(eidasAttributeQueryRequestDto);
+        assertThat(response.getStatus()).isEqualTo(Response.Status.OK.getStatusCode());
```

Comment:
Might be worth digging into that `Response` to assert it's got the country signed stuff in it.

Reply:
Done.

Reply:
I'm not seeing the updated test?